enhancements/template.md

```diff
@@ -193,6 +193,22 @@ enhancement:
 - What changes (in invocations, configurations, API use, etc.) is an existing
   cluster required to make on upgrade in order to make use of the enhancement?
 
+Upgrade expectations:
+- You only need to support `N->N+1` upgrade steps - so, for example, it is
```

Comment:
may want to be more specific here "upgrade steps between minor versions"

and while we are adding this section we should probably clarify in this section the expectations that z-streams must NOT require n->n+1 and must be able to skip forward to any future z-stream update without intermediate updates

Reply:
so probably best to use `Y->Y+1` instead of `N->N+1`

Reply:
Thanks, think I got all of those suggestions